.travis.yml

```diff
@@ -4,6 +4,7 @@ services:
   - docker
 node_js:
   - 8
+  - 10
```

Comment:
I don’t think we need to run the tests in multiple node versions, the fact that the node server is running in these versions should be tested in the `node` package. I’d set the node_js version to `lts/*`

Reply:
okay, thanks. It was a bit of WIP to see if node 10 was causing issues with glitch and now deployment. I need to to a bit more testing locally before PRs.